app/Config/Schema/db_acl.php

```diff
@@ -41,62 +43,62 @@ public function before($event = array()) {
  * @param array $event The event data.
  * @return void
  */
-	public function after($event = array()) {
-	}
+    public function after($event = array())
+    {
+    }
 
 /**
  * ACO - Access Control Object - Something that is wanted
  */
-	public $acos = array(
-		'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
-		'parent_id' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
-		'model' => array('type' => 'string', 'null' => true),
-		'foreign_key' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
-		'alias' => array('type' => 'string', 'null' => true),
-		'lft' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
-		'rght' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
-		'indexes' => array(
-			'PRIMARY' => array('column' => 'id', 'unique' => 1),
-			'idx_acos_lft_rght' => array('column' => array('lft', 'rght'), 'unique' => 0),
-			'idx_acos_alias' => array('column' => 'alias', 'unique' => 0)
-		)
-	);
+    public $acos = array(
+        'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
+        'parent_id' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
+        'model' => array('type' => 'string', 'null' => true),
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 20 spaces but found 8
* Array with multiple values cannot be declared on a single line